frontend/app/src/scenes/Home/index.tsx

```diff
@@ -94,15 +94,22 @@ class Home extends React.Component<HomeProps> {
                   </Button>
                 }
               >
-                <Menu.Item>
-                  <Link to="/dashboard/universities/add/">University</Link>
-                </Menu.Item>
-                <Menu.Item>
-                  <Link to="/dashboard/ambassadors/add">Ambassador</Link>
-                </Menu.Item>
-                <Menu.Item>
-                  <Link to="/dashboard/academix/item/add">Item</Link>
-                </Menu.Item>
+                <Menu.ItemGroup title="Multiverse">
+                  <Menu.Item>
```

Comment:
Add unique keys for each menu item

Reply:
Is it okay to use a string like `add-item` as the key?

Reply:
Resolved with commit [5c9ed7c](https://github.com/sef-global/sef-core/pull/75/commits/5c9ed7cf928bf44cff036b11cd9abb2e3ccf51e4)

Reply:
yes, it's ok. The key is for the item. not for the link

Reply:
Got it!

Reply:
Resolved with commit [8d8d2ca](https://github.com/sef-global/sef-core/pull/75/commits/8d8d2ca1b7aaebafbf573bc0b20e319a0ee04d77)